python/ray/experimental/workflow/tests/test_virtual_actor.py

```diff
@@ -0,0 +1,46 @@
+import ray
+
+from ray.experimental import workflow
+
+
+@workflow.actor
+class Counter:
+    def __init__(self, x: int):
+        self.x = x
+
+    def get(self):
+        return self.x
+
+    def incr(self):
+        self.x += 1
+        return self.x
+
+    def __getstate__(self):
+        return self.x
+
+    def __setstate__(self, state):
+        self.x = state
+
+
+@workflow.step
+def init_virtual_actor(x):
+    return x
+
+
+def test_readonly_actor():
+    ray.init()
+    r = workflow.run(init_virtual_actor.step(42), workflow_id="simple_actor")
+    assert ray.get(r) == 42
+    s = Counter.options(actor_id="simple_actor", readonly=True).get()
+    assert ray.get(s.get.run()) == 42
+    assert ray.get(s.incr.run()) == 43
+    assert ray.get(s.get.run()) == 42
```

Comment:
Should we also re-construct the virtual actor here (reading from storage) and read the value after incr?